application.py

```diff
@@ -3,7 +3,9 @@
 import os
 
 import sentry_sdk
-from ddtrace import patch_all
+# from ddtrace import patch_all
```

Comment:
why is this commented out? if we are not using, please remove. if this is a TODO, please add TODO and the ticket this will be used in.

Reply:
removed!  Thank you